pkg/storage/common/common.go

```diff
@@ -610,6 +610,11 @@ func IsSignature(descriptor ispec.Descriptor) bool {
 			return true
 		}
 
+		// is cosign signature (OCI 1.1 support)
+		if descriptor.ArtifactType == zcommon.ArtifactTypeCosign {
```

Comment:
should we change isCosignSigned to use zcommon.ArtifactTypeCosign similar to isNotationSigned ?

Reply:
fixed